loopchain/baseservice/peer_manager.py

```diff
@@ -110,29 +139,64 @@ def get_peer_ids_hash(peer_ids):
         util.logger.debug(f"peer ids hash({peer_ids_hash})")
         return peer_ids_hash
 
-    @property
-    def peer_list(self) -> dict:
-        """
+    def load_peers_from_iiss(self):
+        util.logger.debug(f"load peers from iiss...")
 
-        :return: { peer_id:PeerInfo }
-        """
-        # util.logger.spam(f"peer_list({self.peer_list_data.peer_info_list})")
-        return self.peer_list_data.peer_info_list
+        request = {
+            "method": "ise_getPRepList"
+        }
 
-    @property
-    def peer_order_list(self) -> dict:
-        """
+        request = convert_params(request, ParamType.call)
+        stub = StubCollection().icon_score_stubs[ChannelProperty().name]
+        response = cast(dict, stub.sync_task().call(request))
+        response_to_json_query(response)
 
-        :return: { order:peer_id }
-        """
-        return self.peer_list_data.peer_order_list
+        util.logger.notice(f"in load_peers_from_iiss response({response})")
+        if not response['result'].get('preps'):
+            util.logger.debug(f"There is no preps in result.")
+            return
 
-    def set_peer_list(self, peer_list_data: PeerListData):
-        """ update PeerList
+        peer_ids = (preps["id"] for preps in response["result"]["preps"])
+        peer_ids_appended = reduce(add, peer_ids, '')
 
-        :param peer_list_data: PeerListData
-        """
-        self.peer_list_data = peer_list_data
+        if self.get_peer_ids_hash(peer_ids_appended) == self.peer_ids_hash():
+            util.logger.debug(f"There is no change in peers.")
+            return
+
+        util.logger.debug(f"Peer manager have to update with new list.")
+        self.reset_peers(check_status=False)
+
+        if not conf.LOAD_PEERS_FROM_IISS:
+            return
+
+        for order, rep_info in enumerate(response["result"]["preps"], 1):
+            peer_info = PeerInfo(rep_info["id"], rep_info["id"], rep_info["target"], order=order)
+            self.add_peer(peer_info)
+        self.show_peers()
+
+    async def load_peers_from_file(self):
+        channel_info = await StubCollection().peer_stub.async_task().get_channel_infos()
+        for peer_info in channel_info[ChannelProperty().name]["peers"]:
+            self.add_peer(peer_info)
+        self.show_peers()
+
+    def show_peers(self):
+        util.logger.debug(f"peer_service:show_peers ({ChannelProperty().name}): ")
+        for peer in self.get_IP_of_peers_in_group():
+            util.logger.debug("peer_target: " + peer)
+
+    def get_quorum(self):
```

Comment:
type hint of return value please.

Reply:
It's not written in this PR, it's just refactoring.